node/pkg/driver/node_utils.go

```diff
@@ -230,11 +238,37 @@ func (n NodeUtils) ParseFCPorts() ([]string, error) {
 	return fcPorts, nil
 }
 
-func (n NodeUtils) Exists(path string) bool {
+func (n NodeUtils) IsPathExists(path string) bool {
 	_, err := os.Stat(path)
 	if err != nil {
+		if !os.IsNotExist(err) {
+			return false
```

Comment:
remove this line

Reply:
why?

Reply:
return it first (line 245), and then print a warning (246), the warning will never be printed.

Reply:
ok, so you need to move the warning out of this if loop.

Reply:
This is basic, there must be a tool that can automatically fail on these kind of mistakes.
@igorgonibm Please open a ticket to strengthen our go validations (like I did with PEP8), before resolving this.

Reply:
@igorgonibm please write the ticket number here before resolving